test cases/failing/80 dub library/test.json

```diff
@@ -1,4 +1,5 @@
 {
+  "expect_skip_on_jobname": ["opensuse", "ubuntu-rolling"],
```

Comment:
Note the name: "expect_skip". 

Not "skip". This doesn't skip tests, it asserts in CI that if they are skipped, it doesn't constitute "UNEXPECTEDSKIP" status.

Reply:
The CI has finished collecting the os_comp jobs now -- this PR causes 7 jobs to fail. e.g

```
 [UNEXRUN]   frameworks: 17 mpi    (method=auto)
Reason: Test ran, but was expected to be skipped for MESON_CI_JOBNAME 'linux-ubuntu-rolling-clang'
 [UNEXRUN]   frameworks: 17 mpi    (method=config-tool)
Reason: Test ran, but was expected to be skipped for MESON_CI_JOBNAME 'linux-ubuntu-rolling-clang'
 [UNEXRUN]   frameworks: 17 mpi    (method=pkg-config)
Reason: Test ran, but was expected to be skipped for MESON_CI_JOBNAME 'linux-ubuntu-rolling-clang'
```

```
 [UNEXRUN]   frameworks: 24 libgcrypt
Reason: Test ran, but was expected to be skipped for MESON_CI_JOBNAME 'linux-arch-gcc'
```